speaker/main.go

```diff
@@ -69,17 +70,19 @@ func main() {
 	}
 
 	var (
-		config      = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
+		configMap   = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
 		configNS    = flag.String("config-ns", "", "config file namespace (only needed when running outside of k8s)")
 		kubeconfig  = flag.String("kubeconfig", "", "absolute path to the kubeconfig file (only needed when running outside of k8s)")
-		host        = flag.String("host", os.Getenv("METALLB_HOST"), "HTTP host address")
+		metricsHost = flag.String("metrics-host", os.Getenv("METALLB_METRICS_HOST"), "Bind address for Prometheus metrics")
+		metricsPort = flag.Int("metrics-port", 80, "Bind port for Prometheus metrics")
 		mlBindAddr  = flag.String("ml-bindaddr", os.Getenv("METALLB_ML_BIND_ADDR"), "Bind addr for MemberList (fast dead node detection)")
 		mlBindPort  = flag.String("ml-bindport", os.Getenv("METALLB_ML_BIND_PORT"), "Bind port for MemberList (fast dead node detection)")
 		mlLabels    = flag.String("ml-labels", os.Getenv("METALLB_ML_LABELS"), "Labels to match the speakers (for MemberList / fast dead node detection)")
 		mlNamespace = flag.String("ml-namespace", os.Getenv("METALLB_ML_NAMESPACE"), "Namespace of the speakers (for MemberList / fast dead node detection)")
 		mlSecret    = flag.String("ml-secret-key", os.Getenv("METALLB_ML_SECRET_KEY"), "Secret key for MemberList (fast dead node detection)")
 		myNode      = flag.String("node-name", os.Getenv("METALLB_NODE_NAME"), "name of this Kubernetes node (spec.nodeName)")
-		port        = flag.Int("port", 80, "HTTP listening port")
+		statusHost  = flag.String("status-host", os.Getenv("METALLB_STATUS_HOST"), "Bind address for status endpoint")
+		statusPort  = flag.Int("status-port", 8080, "Bind port for status endpoint")
```

Comment:
The docs don't actually say what the default is, but rather assume the port is `7473` because that's what is used in the manifest.

I've copied the behavior from the metrics endpoint, although I agree that it's a bit strange that the code sets a default and the manifest overrides it.

How would you handle this?

Reply:
I'd just make the default `7473` here.